docs/rest_api_2.rst

```diff
@@ -0,0 +1,4156 @@
++=============+================================================================================================================================+=========================================================================================================================================================+
+| ldev        | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device>                                                      |     uni/tn-Sample\_2/lDevVip-f5-gs                                                                                                                      |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the <Your L4-L7 Device> and click on “Open in Object Store Browser”                                          |                                                                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 3. Use the dn property from the object browser for input parameter “ldev”.                                                     |                                                                                                                                                         |
++-------------+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| lif         | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device> → Cluster Interfaces→ <Your Interface of Interest>   |     uni/tn-Sample\_2/lDevVip-f5-gs/lIf-external                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the logical interface and click on “Open in Object Store Browser”                                            |                                                                                                                                                         |
```

Comment:
remove "on" in both places and replace "" with ** **

Reply:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)